app/models/concerns/foreman_build_history/host_extensions.rb

```diff
@@ -7,6 +7,7 @@ module HostExtensions
       after_build :set_build_start
       before_provision :set_build_finish
       before_save :build_history_handle_host_creation
+      has_one :build_history_facet, :foreign_key => 'host_id'
```

Comment:
First, thanks for looking into this plugin, I really appreciate it!

This change is unnecessary though, facet framework handles facet relations [automatically](https://github.com/theforeman/foreman/blob/develop/app/models/concerns/facets/managed_host_extensions.rb#L38)